2021-10-28 21:32:39,002 INFO    MainThread:20757 [wandb_setup.py:_flush():71] setting env: {}
2021-10-28 21:32:39,002 INFO    MainThread:20757 [wandb_init.py:_log_setup():357] Logging user logs to /home/siddhi/workspace/ColorCapsNet/wandb/offline-run-20211028_213239-15o2cwzs/logs/debug.log
2021-10-28 21:32:39,002 INFO    MainThread:20757 [wandb_init.py:_log_setup():358] Logging internal logs to /home/siddhi/workspace/ColorCapsNet/wandb/offline-run-20211028_213239-15o2cwzs/logs/debug-internal.log
2021-10-28 21:32:39,002 INFO    MainThread:20757 [wandb_init.py:init():390] calling init triggers
2021-10-28 21:32:39,002 INFO    MainThread:20757 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-28 21:32:39,002 INFO    MainThread:20757 [wandb_init.py:init():435] starting backend
2021-10-28 21:32:39,002 INFO    MainThread:20757 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-28 21:32:39,007 INFO    MainThread:20757 [backend.py:ensure_launched():198] starting backend process...
2021-10-28 21:32:39,011 INFO    MainThread:20757 [backend.py:ensure_launched():203] started backend process with pid: 20769
2021-10-28 21:32:39,012 INFO    MainThread:20757 [wandb_init.py:init():444] backend started and connected
2021-10-28 21:32:39,013 INFO    MainThread:20757 [wandb_init.py:init():503] updated telemetry
2021-10-28 21:32:39,021 INFO    MainThread:20757 [wandb_init.py:init():576] starting run threads in backend
2021-10-28 21:32:41,190 INFO    MainThread:20757 [wandb_run.py:_console_start():1693] atexit reg
2021-10-28 21:32:41,191 INFO    MainThread:20757 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-10-28 21:32:41,191 INFO    MainThread:20757 [wandb_run.py:_redirect():1572] Redirecting console.
2021-10-28 21:32:41,191 INFO    MainThread:20757 [wandb_run.py:_redirect():1628] Redirects installed.
2021-10-28 21:32:41,192 INFO    MainThread:20757 [wandb_init.py:init():603] run started, returning control to user process
2021-10-28 21:34:54,799 INFO    MainThread:20757 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-10-28 21:34:55,586 INFO    MainThread:20757 [wandb_run.py:_restore():1635] restore
2021-10-28 21:35:04,046 INFO    MainThread:20757 [wandb_run.py:_wait_for_finish():1793] got exit ret: 
2021-10-28 21:35:04,526 INFO    MainThread:20757 [wandb_run.py:_wait_for_finish():1793] got exit ret: 
2021-10-28 21:35:04,772 INFO    MainThread:20757 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
local_info {
}

